src/System.Linq/tests/EnumerableTests.cs

```diff
@@ -1,178 +1,291 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Collections;
 using System.Collections.Generic;
 using Xunit;
+using Xunit.Extensions;
 
 namespace System.Linq.Tests
 {
-    public partial class EnumerableTests
+    public class EnumerableTests
```

Comment:
Consider making this abstract to highlight that it's meant to be used only as a base class.